packages/next/build/webpack-config.ts

```diff
@@ -160,6 +166,20 @@ export default async function getBaseWebpackConfig(
       next: NEXT_PROJECT_ROOT,
       [PAGES_DIR_ALIAS]: pagesDir,
       [DOT_NEXT_ALIAS]: distDir,
+      ...(!isServer
+        ? {
+            unfetch: path.join(__dirname, 'polyfills', 'fetch.js'),
+            'isomorphic-unfetch': path.join(__dirname, 'polyfills', 'fetch.js'),
+            'whatwg-fetch': path.join(__dirname, 'polyfills', 'fetch.js'),
+            'object-assign': path.join(
```

Comment:
will this get re-used here: https://github.com/azukaru/next.js/pull/142/files#diff-dd19f84f6905f5f026b78648a481f0b5R2?

Reply:
Yup! Fixed this.